lib/src/pubspec.dart

```diff
@@ -112,6 +112,21 @@ class Pubspec {
 
   SdkConstraintStatus get sdkConstraintStatus =>
       SdkConstraintStatus.fromSdkVersion(_inner.environment['sdk']);
+
+  /// The sdk tags associated with [this].
+  List<String> sdkTags() {
+    final result = <String>[];
+    final dependentSdks = this.dependentSdks;
+    if (dependentSdks.every(['dart'].contains)) {
+      result.add('sdk:dart');
+    }
+    if (dependentSdks.every(['dart', 'flutter'].contains)) {
```

Comment:
Same here:
```dart
if (dependentSdks.length == 2 && dependentSdks.containsAll(['dart', 'flutter']) {
```

Reply:
No - not exactly the same.
the expression i wrote is true for `dependentSdks = ['dart']` (You get the 'sdk:flutter' tag as soon as you can be used in flutter, not if you require flutter).
I agree that it is cryptic, and would be happy to write it in a better way.

Reply:
I see, makes sense. Please add a code comment with this information, so that we'll remember it a few months down the line :).

Reply:
Good idea! Thanks